scripts/globals/abilities/feather_step.lua

```diff
@@ -0,0 +1,180 @@
+-----------------------------------
+-- Ability: Feather Step
+-- Description Lowers a target's critical hit evasion. If successful, will earn you a finishing move.
+-- Obtained: DNC Level 83
+-- Recast Time: 00:00:05 (Step)
+-- Duration: 00:01:00
+-- Cost: 100TP
+-----------------------------------
+require("scripts/globals/weaponskills")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+
+function onAbilityCheck(player, target, ability)
+    if (player:getAnimation() ~= 1) then
+        return tpz.msg.basic.REQUIRES_COMBAT, 0
+    else
+        if (player:hasStatusEffect(tpz.effect.TRANCE)) then
+            return 0, 0
+        elseif (player:getTP() < 100) then
+            return tpz.msg.basic.NOT_ENOUGH_TP, 0
+        else
+            return 0, 0
+        end
+    end
+end
+
+function onUseAbility(player, target, ability, action)
+    -- Only remove TP if the player doesn't have Trance.
+    if not player:hasStatusEffect(tpz.effect.TRANCE) then
+        player:delTP(100)
+    end
+
+    local hit = 2
+    local effect = 1
+
+    if math.random() <= getHitRate(player, target, true, player:getMod(tpz.mod.STEP_ACCURACY)) then
+        hit = 6
+        local mjob = player:getMainJob()
+        local daze = 1
+
+        if (mjob == 19) then
```

Comment:
mjob called once in this script, it can likely be removed and this if check replaced with 
``if player:getMainJob() == 19 then``

Reply:
removed this

Reply:
Not `== 19` never use the number.

Use `== tpz.job.DNC` instead see https://github.com/project-topaz/topaz/blob/release/scripts/globals/status.lua#L54 for further info

Reply:
Oh Man was I so dumb this morning looking at this I was tryin to rush n not really look... lol .... sorry will fix when I’m back at the pc